mithril.js

```diff
@@ -593,7 +593,7 @@ var renderService = function($window) {
 				}
 			}
 			if (vnode.dom.parentNode != null) parent.removeChild(vnode.dom)
-			if (context != null && vnode.domSize == null && !hasIntegrationMethods(vnode.attrs) && typeof vnode.tag === "string") { //TODO test custom elements
+			if (context != null && vnode.domSize == null && !hasIntegrationMethods(vnode.attrs) && typeof vnode.tag === "string" && vnode.tag !== "<") { //TODO test custom elements
```

Comment:
Actually, it corresponds to the [current tip](https://github.com/lhorie/mithril.js/blob/608a494538daede4e16c6e02114e36366d2d49d5/render/render.js#L359) of the rewrite branch (post #1185 fix).

CI failed because of a flaky test (not the first time I get a spurious failure in an async one).

```
throttle > calls after threshold: 1 should equal 2

    at Timeout._onTimeout (/home/travis/build/lhorie/mithril.js/api/tests/test-throttle.js:42:21)
```